lib/ship_compliant/client.rb

```diff
@@ -6,7 +6,7 @@ class << self
   # Returns an instance of +Client+.
   def self.client(configuration: :default)
     config = get_config_object_for_key(configuration)
-    self.ship_compliant_client ||= new_client_from_wsdl(config.wsdl,
+    self.ship_compliant_client = new_client_from_wsdl(config.wsdl,
```

Comment:
Yes, but how else can we dynamically pass in the config type, if it is something other than `:default`?